### PR TITLE
Update react docs & samples with new packages

### PR DESCRIPTION
**Pull request by tnorling**
- Updates references to `msal-browser` and `msal-react` to `@azure/msal-browser` and `@azure/msal-react`
- Updates samples to use the new beta package
- Updates build/test workflow to not run when changefiles are added

### CODE REVIEW

**Review by jo-arroyo — APPROVED** [review]

File: .github/workflows/build-steps.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ on:
       - '**.md'
       - '**/.vscode/*.json'
       - 'samples/**'
+      - 'change/**'
```
Comment (pkanher617) [inline .github/workflows/build-steps.yml:16]: Are we sure we want to ignore change-files? If there is a new change-file shouldn't we assume that a build should be run?

Comment (tnorling) [reply .github/workflows/build-steps.yml:16]: The presence of a changefile on its own should not trigger a build. The presence of changed files in the libs/samples/workflow should be what triggers a new build

Comment (tnorling) [reply .github/workflows/build-steps.yml:16]: Sorry, not samples in this case, but in E2E case

Comment (pkanher617) [reply .github/workflows/build-steps.yml:16]: Makes sense

**Comment by pkanher617** [comment]
We should make sure to update the main page README and update any references to alpha to beta (for both React and Angular)

**Review by pkanher617 — APPROVED** [review]
One comment about references to `alpha`, otherwise looks good.

**Pull request merged**